### DISCUSSION TITLE
Detect when/if AIVE-steered combine is turning

### DISCUSSION THREAD

**Discussion by Stephan-S** [Mogli12/AIVehicleExtension#266] [Ideas]
Hello Stefan :-)

Stephan here from the AD mod.
Could you point me to the correct variable/state to check to detect when an AIVE-driven combine is actually turning or just following the crop line with it's wheels turned.
In the former case, AD should stop and wait for the combine to finish the turn while in the latter, I want to keep chasing the pipe.

And if you like, I would gladly have you join our Discord so questions like this could be asked and answered in a more direct and personal way, rather then creating (non)-issues just to open a communication channel :D

Here is a link to the AutoDrive Discord: https://discord.gg/9D4B5hX
If you have one I could join, let me know.

Thanks and have a nice easter holiday despite the current times.

**Comment by Mogli12** [comment]
Hello,

I think you should use self.spec_aiVehicle.isTurning Please check line 835 of [AIVehicle](https://gdn.giants-software.com/documentation_scripting_fs19.php?version=script&category=70&class=10462#aiEndTurn165123)

Regards, Stefan
